src/server/configexpress.js

```diff
@@ -19,16 +19,12 @@ class ConfigExpress {
         this._setRouters();
     }
 
-    set (key, value) {
-        this.express.set(key, value);
-    }
-
     _setMiddlewares () {
         this.express.use(logger);
     }
 
     _setVariables () {
-        this.express.set('json spaces', 2);
+        this.express.set('json spaces', config.JSON_SPACES);
```

Comment:
It's better to introduce a constant for `json spaces` too.

Reply:
But this constant will not belong to our app configuration, because it's [express application setting constant](https://expressjs.com/en/api.html#app.settings.table). So, does it make sense?